src/utils/SVG.cpp

```diff
@@ -210,7 +285,45 @@ void SVG::writePolygon(ConstPolygonRef poly, Color color, float stroke_width)
     int i = 0;
     for (Point p1 : poly)
     {
-        if (color == Color::RAINBOW)
+        if (color.color == Color::RAINBOW)
+        {
+            int g = (i * 255 * 11 / size) % (255 * 2);
+            if (g > 255) g = 255 * 2 - g;
```

Comment:
Brackets on new line.

```suggestion
            if (g > 255)
            {
                g = 255 * 2 - g;
            }
```